### PR TITLE
feat(root certs) solves many issues with tls module, http module and fetch

### PR DESCRIPTION
**Pull request by cirospaciari**

### CODE REVIEW

File: src/crypto/root_certs.h [hunk 1]
```diff
@@ -0,0 +1,3365 @@
+static const char* const root_certs[] = {
```
Comment (Jarred-Sumner) [inline src/crypto/root_certs.h:27]: where are these from?

Comment (cirospaciari) [reply src/crypto/root_certs.h:27]: From Mozilla certdata.txt. nodejs and dart do the same we will need a update process for each release probably. I will add a comment on how I generated it and create a script

Comment (Jarred-Sumner) [reply src/crypto/root_certs.h:27]: yeah this needs to be a big comment at the top explaining where the list was generated

also this is sensitive stuff from a security perspective

we probably should move our fork of uSockets to be more directly in bun's repo or at least under oven-sh

Comment (Jarred-Sumner) [reply src/crypto/root_certs.h:27]: not right now but its something to consider

Comment (cirospaciari) [reply src/crypto/root_certs.h:27]: @Jarred-Sumner updated

**Pull request merged**